src/CommunityToolkit.Maui.BindablePropertySG/Models/Records.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CommunityToolkit.Maui.BindablePropertySG.Helpers;
+using Microsoft.CodeAnalysis;
+
+namespace CommunityToolkit.Maui.BindablePropertySG.Models;
+
+record BPInfo
```

Comment:
The `IsExternalInit` class is used on our regular SG to allow the usage of `record`s that are great for keeping the incremental behavior. I didn't added as link because VS complained sometimes